### DISCUSSION TITLE
Mad Catz Arcade FightStick TES+  PS3 & PS4

### DISCUSSION THREAD

**Discussion by Mr-Cyb3rnator** [cathery/sys-con#185] [Controller requests]
Description of the controller

USB Arcade stick with support for PS3 and PS4, it doesn't have bluetooth, motion controls, rumble, audio jack

Screenshot of the device

First part PS4 mode, second part PS3 mode

![2020052521101800-E7260330E4B7D47C63FF99BA9689D77C](https://user-images.githubusercontent.com/65927173/82857253-b9264a00-9ecd-11ea-8781-b5665ed6cec2.jpg)

Additional context
https://www.amazon.com/-/es/dp/B018AW408C
https://images-na.ssl-images-amazon.com/images/I/91yXX4vyI3S.pdf

**Comment by Panic-** [comment]
+1
